tools/CACertificates/ca-certs.ps1

```diff
@@ -750,12 +755,58 @@ function New-CACertsDevice([Parameter(Mandatory=$TRUE)][string]$deviceName, [str
         CA certificate is used.
 #>
 function New-CACertsEdgeDevice([Parameter(Mandatory=$TRUE)][string]$deviceName, [string]$issuerPrefix=$_intermediatePrefix)
+{
+    if ([string]::IsNullOrWhiteSpace($deviceName))
```

Comment:
Because `$deviceName` is a mandatory parameter, it already can't be null or empty. So that only leaves whitespace, which could be handled with the attribute `[ValidatePattern("[^\s]+")]`.

Reply:
I prefer a friendly error message even though the null check is redundant.